### PR TITLE
Display Autopush on Update page

### LINKED ISSUE #999: Display whether autopush is enabled or disabled on the Update page
With the 2.2.0 release, all updates have karma thresholds displayed, whether or not autopush is enabled. Users are used to looking for the karma levels to decipher whether an update is set to autopush or not, and so users seem to be confused about whether updates autopush.

I propose that we display on the update page whether autopush is or is not enabled so that it is clear what should be expected of Bodhi when karma is given.

### LINKED ISSUE #1032: All my new updates are filed with autokarma enabled even though I unchecked the checkbox
Whenever I file a new update, even if I uncheck the autokarma checkbox, the update is filed with autokarma enabled. I have read issue #999, which claims that this is just a display thing:

> With the 2.2.0 release, all updates have karma thresholds displayed, whether or not autopush is enabled.

but I cannot confirm this: When I edit the update, the checkbox is on! If I then uncheck the checkbox and confirm the edit, the thresholds vanish, and any further edits will no longer show autokarma as enabled.

(Note that the thresholds vanishing when I edit updates to disable autokarma also means that the improvement from issue #772 (manual push to stable based on karma) does not work, so I am filing a new issue for that.)

### PR DESCRIPTION
**Pull request by trishnaguha**
Fixes #999 . This might also fix #1032 by displaying autopush on the UI. 
Screenshot uploaded:

screenshot from 2016-10-13 17-13-55

### CODE REVIEW

**Comment by trishnaguha** [comment]
@bowlofeggs Does this look good (Displaying `True/False`) ? or do we want to display `Enabled/Disabled` specifically?

File: bodhi/server/templates/update.html
Comment (bowlofeggs) [inline bodhi/server/templates/update.html]: Maybe one more space on this line so it visually lines up.

**Review by bowlofeggs — COMMENTED** [review]
@trishnaguha I think this is acceptable as is, but if you want to go for the gold medal I think writing `Enabled/Disabled` with green and red backgrounds consistent with the type and suggestion labels in your screenshot would be pretty sweet.

Let me know if you want to "go for the gold" in this PR or if you want to just merge this as is and we can change it later.

My space thing is really just a nitpick, but if you want to fix that too that would be nice (not required) ☺

**Comment by trishnaguha** [comment]
@bowlofeggs I'll go for gold ;).

File: bodhi/server/templates/update.html
Comment (trishnaguha) [reply bodhi/server/templates/update.html]: yup my mistake :)

**Comment by trishnaguha** [comment]
Here is the screenshot. We are dynamically displaying `Autopush` on update page.
screenshot from 2016-10-19 13-14-19

**Comment by trishnaguha** [comment]
Before we merge this PR we need to fix #1048

**Comment by trishnaguha** [comment]
This is how it looks like :)

screenshot from 2016-10-19 19-33-17

**Comment by bowlofeggs** [comment]
This is beautiful @trishnaguha! Adding a test would be nice too if you have time. I'm going to hold off on merging this until I fix #1048.

**Comment by trishnaguha** [comment]
@bowlofeggs Ah sure. I'll make sure to add test for it :)

**Comment by bowlofeggs** [comment]
https://github.com/fedora-infra/bodhi/pull/1050 must be merged before this can be merged.

**Comment by trishnaguha** [comment]
Updated the PR with testcases.

**Review by bowlofeggs — APPROVED** [review]
I just hand tested this and it looks really nice!

**Comment by trishnaguha** [comment]
Thanks @bowlofeggs :-)

**Pull request merged**
